src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -2202,6 +2202,8 @@ private void DynamoViewModelRequestShowPackageManager(object s, EventArgs e)
                     WindowStartupLocation = WindowStartupLocation.CenterOwner
                 };
 
+                dynamoViewModel.Owner = packageManagerWindow;
```

Comment:
It seems odd that the dynamoViewModel.Owner is set to pmwindow?

Reply:
I am sure there was a reason for that, but at the moment I am not able to reproduce the behavior that it I thought it was resolving, I will take it out for now, and will keep an eye on any issues with centering prompts (which I think is what this was all about). Just for the record, at this point, the value of `dynamoViewModel.Onwer` is null, not sure if that's relevant.